docs/cloud/getting-started-with-airbyte-cloud.md

```diff
@@ -26,13 +26,10 @@ A source is an API, file, database, or data warehouse that you want to ingest da
 
 To set up a source:
 
-1. On the Airbyte Cloud dashboard, click **Sources** in the left navigation bar. 
-2. In the top right corner, click **+ new source**. 
-3. On the Set up the source page, enter a name for your source.
-4. From the Source type dropdown, select the source you want to set up.
-5. The fields relevant to your source are displayed. 
-Click **Setup Guide** for help with filling in the fields for your selected source.
-6. Click **Set up source**. 
+1. On the Airbyte Cloud dashboard, click **Sources** and then click **+ New source**. 
+2. On the Set up the source page, select the source you want to set up from the **Source type** dropdown.
+3. The fields relevant to your source are displayed. The Setup Guide provides information to help you fill out the fields for your selected source.
```

Comment:
nit: Step 3 isn't really a step since there's no action item for the user. You can merge it with the previous step by adding a newline with the <br> tag.